packages/grid/x-data-grid-pro/src/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -252,6 +253,12 @@ export const useGridColumnResize = (
           `[data-field="${colDef.field}"]`,
         )!;
 
+      colGroupingElementRef.current = Array.from(
```

Comment:
We can move it to `findGridCellElementsFromCol`

Reply:
Not really, `findGridCellElementsFromCol` is used for 2 reasons:
- resize the div (no problem here to move to an array)
- get the current size of the element to compute the width difference before/after (here we need to keep only one result from `findGridCellElementsFromCol`)

Reply:
Right, makes sense!